sandy/core/endf6.py

```diff
@@ -266,8 +271,17 @@ def get_endf6_file(library, kind, zam, to_file=False):
         text = "".join([foo_read(name, url) for name in files.values()])
         tape = Endf6.from_text(text)
     else:
-        filename = files[zam]
-        tape = foo_get(filename, url)
+        if type(zam) == pd.Series:
+            zam = zam.values()
+        if zam.__len__() != 1:
```

Comment:
@EnricaBelfiore, you shouldn't approve this! This will crash if zam is an integer.
@AitorBengoechea , I am surprised you don't get errors...probably because you don't have any test where you ask for a single decay nuclide (only all or list). This test should be added, otherwise these mistakes will not be detected.

Reply:
The tests for individual nucleus are put inside the get_endf6_file function, but even passing pytest with the mentioned error, pytest still pass the tests.
The program breaks in __FormattedFile.merge, i do not know why